templates/contact-us.html

```diff
@@ -5,12 +5,12 @@
     <div class="p-strip--suru-image">
       <div class="row">
         <div class="col-6">
-          <img src="https://assets.ubuntu.com/v1/6685f10c-contact-canonical-icon.svg" alt="Canonical contact icon">
+          <img src="https://assets.ubuntu.com/v1/6685f10c-contact-canonical-icon.svg" alt="Contact Canonical">
```

Comment:
There are no actual headings whatsoever on this page, making it harder to parse on screen readers. This header strip could be:
```suggestion
          <h1 class="u-off-screen">Contact Canonical</h1>
          <img src="https://assets.ubuntu.com/v1/6685f10c-contact-canonical-icon.svg">
```